client/src/features/visualizer-threejs/wrapper/StatsPanel.tsx

```diff
@@ -1,27 +1,14 @@
 import React from "react";
 import { useNetworkStats } from "~helpers/stardust/hooks/useNetworkStats";
 
-export const StatsPanel: React.FC<{ readonly blocksCount: number; readonly network: string }> = ({ blocksCount, network }) => {
-    const [blocksPerSecond, confirmedBlocksPerSecond, confirmedBlocksPerSecondPercent] = useNetworkStats(network);
-
+export const StatsPanel = ({ network }: { network: string }) => {
+    const [blocksPerSecond] = useNetworkStats(network);
```

Comment:
This should use a hook for nova (from `..../helpers/nova/hooks`). The one from stardust maybe works, but we should have a separate one for nova.